src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -223,7 +223,18 @@ bool Terminal::SendKeyEvent(const WORD vkey,
     // KeyEvent.
     // DON'T manually handle Alt+Space - the system will use this to bring up
     // the system menu for restore, min/maximimize, size, move, close
-    wchar_t ch = altPressed && vkey != VK_SPACE ? static_cast<wchar_t>(LOWORD(MapVirtualKey(vkey, MAPVK_VK_TO_CHAR))) : UNICODE_NULL;
+    wchar_t ch = UNICODE_NULL;
+    if (altPressed && vkey != VK_SPACE)
+    {
+        ch = static_cast<wchar_t>(LOWORD(MapVirtualKey(vkey, MAPVK_VK_TO_CHAR)));
+        // MapVirtualKey will give us the capitalized version of the char.
+        // However, if shift isn't pressed, we want to send the lowercase version.
+        // (See GH#637)
+        if (!shiftPressed && (ch >= L'A' && ch <= L'Z'))

```

Comment:
`S C A R Y`

What does `Alt+I` turn into on a Turkish locale? Should it be `\x1bı` (`LATIN SMALL LETTER DOTLESS I`), or should it be `\x1bi` (`LATIN SMALL LETTER I`)?

How about if they press `Alt+İ` (`LATIN CAPITAL LETTER I WITH DOT ABOVE`)?

Reply:
I choose Turkish specifically because it is one example where `i` does not capitalize to `I` and `I` does not lowercase to `i`.